testData/solutions/linux.json5

```diff
@@ -1289,34 +1289,122 @@
                     "path": "net.gpii.uioPlus"
                 },
                 "supportedSettings": {
-                    "lineSpace": {},
-                    "fontSize": {},
-                    "characterSpace": {},
-                    "inputsLargerEnabled": {},
-                    "contrastTheme": {},
-                    "selfVoicingEnabled": {},
-                    "selectionTheme": {},
-                    "tableOfContentsEnabled": {},
-                    "simplifiedUiEnabled": {},
-                    "syllabificationEnabled": {},
-                    "captionsEnabled": {}
-                },
-                "capabilitiesTransformations": {
-                    "lineSpace": "http://registry\\.gpii\\.net/common/lineSpace",
+                    "captionsEnabled": {
+                        "schema": {
+                            "title": "Captions",
+                            "description": "Request videos to display captions.",
+                            "type": "boolean",
+                            "default": false
+                        }
+                    },
+                    "characterSpace": {
+                        "schema": {
+                            "title": "Character Space",
+                            "description": "Adjust the space between characters.",
+                            "type": "number",
+                            "default": 1,
+                            "minimum": 0.9,
+                            "maximum": 2,
+                            "multipleOf": 0.1
+                        }
+                    },
+                    "contrastTheme": {
+                        "schema": {
+                            "title": "Contrast Theme",
+                            "description": "Change text and background colors.",
+                            "type": "string",
```

Comment:
This type information is redundant for an enum.  The only small danger is that the validation output would include two failures if you (for example) passed a number, as it's neither a string nor one of the accepted values.  Although it's mostly harmless, I'd argue for leaving it out as it makes for simpler feedback.  Copying @amb26 to get his opinion on this small bit of redundancy.

Reply:
I'm happy for this to be omitted